src/pro/sky/java/course1/Employee.java

```diff
@@ -0,0 +1,55 @@
+package pro.sky.java.course1;
+
+public class Employee {
+    static int cnt = 0;
+    private int id;
```

Comment:
```suggestion
    static int cnt = 0;
    
    private int id;
```
часто для читаемости static отделяют пустой строкой от обычных полей